lib/hacluster.pm

```diff
@@ -626,7 +626,9 @@ sub ha_export_logs {
     script_run "touch $corosync_conf";
     script_run "crm report $report_opt -E $bootstrap_log $crm_log", 300;
     upload_logs("$bootstrap_log", failok => 1);
-    upload_logs("$crm_log.tar.bz2", failok => 1);
```

Comment:
`upload_logs("$crm_log*", failok => 1);` doesn't work. It will report error `(26) fail to open/read local data from ...`, and for another reason, when calling `upload_logs` without specify `log_name =>` parameter, it will use the source file's basename as the upload file name. If we pass `log_name`, we also need to know the $crm_log suffix (tar.bz2 or tar.gz). So I keep `script_output` here.

Reply:
Oh.  Then I guess either the API calling `curl` to upload the log, or `curl` itself is not able to expand `*`. Disregard my suggestion ... it was a bad one. :)